src/Microsoft.VisualStudio.Editor.Razor/Microsoft.VisualStudio.Editor.Razor.csproj

```diff
@@ -7,9 +7,9 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Features" Version="$(RoslynDevVersion)" NoWarn="KRB4002" />
```

Comment:
Same `NoWarn` question on all of these.

Reply:
Right, NoWarn="KRB4002" isn't needed anymore because we aren't using package lineup. KRB4002 was our own custom warning for packages that had a version on them even though lineups were used. That doesn't apply now.